inject.py

```diff
@@ -5,53 +5,16 @@
 
 # send({cmd: "path", params: {start: {x: 1, y: 2, z: 3}, end: {x: 1, y: 2, z: 3}}})
 
-def pathsolver_init():
```

Comment:
I'll move this code into a branch, I might come back to it one day...but it's not complete enough to be useful today 😆